ui/src/config/router.js

```diff
@@ -221,9 +221,17 @@ export function asyncRouterMap () {
       generateRouterMap(offering),
       generateRouterMap(config),
       generateRouterMap(tools),
+      {
```

Comment:
do we need it defined here or can it be in an existing section or create a new section (already see infra/usage.js)?